agent/selfupdater/updater_native.go

```diff
@@ -3,26 +3,6 @@
 
 package selfupdater
 
-import (
-	"github.com/Masterminds/semver"
-)
-
-type nativeUpdater struct {
-	version string
-}
-
-func (n *nativeUpdater) CurrentVersion() (*semver.Version, error) {
-	return semver.NewVersion(n.version)
-}
-
-func (n *nativeUpdater) ApplyUpdate(_ *semver.Version) error {
-	return nil
-}
-
-func (n *nativeUpdater) CompleteUpdate() error {
-	return nil
-}
-
 func NewUpdater(version string) (Updater, error) {
```

Comment:
As this file is kept, might make sense to keep the methods here.

Reply:
> As this file is kept, might make sense to keep the methods here.

See the build tags to understand why the methods were moved to another file.

Reply:
I mean, looking only at the diff it doesn't appear to do what I had in mind

Reply:
Sorry but it isn't clear to me. Please provide the link where I see it.

Reply:
https://github.com/shellhub-io/shellhub/blob/97881681d4294204c63240eee73f8b01a7c3ff05/agent/selfupdater/updater_docker.go#L1-L4

https://github.com/shellhub-io/shellhub/blob/97881681d4294204c63240eee73f8b01a7c3ff05/agent/selfupdater/updater_docker.go#L223-L234

https://github.com/shellhub-io/shellhub/blob/97881681d4294204c63240eee73f8b01a7c3ff05/agent/selfupdater/updater_native.go#L1-L8

Reply:
Ok; sure. But the native struct has been moved to `updater.go` and it seems to belong in `updater_native.go`.

Reply:
There is a two `NewUpdater` func that are used according to the build tags. That's why I need to keep the `updater_native.go` file.

Reply:
> Ok; sure. But the native struct has been moved to `updater.go` and it seems to belong in `updater_native.go`.

Yup, the struct has been moved to `updater.go` cause we need them both in `NewUpdater` metods (w/o docker)